mk/spksrc.spk.mk

```diff
@@ -437,19 +437,21 @@ spkclean:
 	       work-*/.depend_done \
 	       work-*/.icon_done \
 	       work-*/.strip_done \
-	       work-*/.wheel_done \
 	       work-*/conf \
 	       work-*/scripts \
 	       work-*/staging \
 	       work-*/tc_vars.mk \
 	       work-*/tc_vars.cmake \
-	       work-*/wheelhouse \
 	       work-*/package.tgz \
 	       work-*/INFO \
 	       work-*/PLIST \
 	       work-*/PACKAGE_ICON* \
 	       work-*/WIZARD_UIFILES
 
+wheelclean: spkclean
+	rm -fr work-*/.wheel_done \
+	       work-*/wheelhouse
```

Comment:
Nice one, would benefit to add `work-*/install/**/share/wheelhouse/*` to clean-up the wheel installation path.